docs/CRSF.md

```diff
@@ -2,36 +2,41 @@
 
 ([back to main page](../README.md))
 
-This page describes how to set up a mLRS system for EdgeTx/OpenTx radios, such that you get the usual CRSF telemetry sensors and can use the Yaapu telemetry app.
+This page describes how to set up a mLRS system for EdgeTX/OpenT radios, so that you get the usual CRSF telemetry sensors and can use the Yaapu telemetry app.
 
-Two things need to be done, (1) the mLRS Tx module needs to be put into "CRSF mode", and (2) the flight controller needs to be set up to send a MAVLink stream with the desired MAVLink messages.
+Two things need to be done:
+1. The mLRS Tx module needs to be put into "CRSF mode"
+2. The flight controller needs to be set up to send a MAVLink stream with the desired MAVLink messages
 
-In principle, there is no specific configuration of the mLRS receiver neccessary. It is however recommended to set the receiver into "mavlink mode", as described below.
+Optional but recommended steps:
+- Set the receiver into "mavlink mode" (described below)
+- Install the LUA script on the radio which allows you to change the Tx and Rx parameters: ([LUA Script](LUA.md))
 
-Note: Any radio which supports CRSF should work. This should include many brands besides EdgeTx/OpenTx radios.
-
-Note: An ArduPilot flight controller is assumed. For PX4 and iNav it needs to be tested and seen.
+Notes:
+- Any radio which supports the CRSF protocol should work, this includes many brands besides EdgeTX/OpenTX radios.
+- An ArduPilot flight controller is assumed. PX4 and INAV needs to be tested and validated.
 
 
 ## mLRS Tx Module Setup
 
 - Tx Ch Source = crsf
+- TX Ser Baudrate = 57600
```

Comment:
not required, move a comment to notes?

Reply:
Moved to notes section, added recommended 57600 as that works on all operation modes.